op-guide/configuration.md

```diff
@@ -50,3 +50,152 @@ The default TiDB ports are 4000 for client requests and 10080 for status report.
 ### $TIDB_PPROF environment variable
 + An environment variable that is used to enable or disable the runtime profiling data via the HTTP server. The Address is at client URL + "/debug/pprof/".
 + If set $TIDB_PPROF to 0, TiDB will disable pprof. Otherwise TiDB will enable pprof.
+
+## PD
```

Comment:
Applies to all flags: 
1. The -> the
2. Remove the full stop "." at the end of the segment.